ggu/main.c

```diff
@@ -582,7 +583,8 @@ run_ggu(struct net_config *net_conf,
 	ggu_conf->ggu_dst_port = rte_cpu_to_be_16(ggu_conf->ggu_dst_port);
 
 	ret = init_mailbox("ggu_mb", MAILBOX_MAX_ENTRIES,
-		sizeof(struct ggu_request), ggu_conf->lcore_id,
+		sizeof(struct ggu_request) + gk_conf->back_max_pkt_burst *
+		sizeof(struct rte_mbuf *), ggu_conf->lcore_id,
```

Comment:
We are mixing things up here. `gk_conf->back_max_pkt_burst` is being used here as a parameter of GGU's mailbox, instead of how many packets the GGU block should read from the back interface. We need a parameter for the mailbox here.